libs/defi/oeth/src/redeem/components/RedeemActionCard.tsx

```diff
@@ -50,15 +50,15 @@ export const RedeemActionCard = ({
     data: swapGasPrice,
     isLoading: swapGasPriceLoading,
     isFetching: swapGasPriceFetching,
-  } = useGasPrice(estimatedRoute?.gas, {
+  } = useGasPrice(estimatedRoute?.gas, undefined, {
```

Comment:
Wasn't sure what this `undefined` was so I looked it up. I think I prefer it to not be optional.

Reply:
yep, I'm not happy either about this api, I'll update

Reply:
I made some changes here, take a look at them before you start something

Reply:
yeah, I saw, that is one option, I wanted to refactor those custom hooks api to use a single object arg as it offers more flexibility when updating inputs. This will be part of a future update